packages/nc-gui/pages/signup/[[token]].vue

```diff
@@ -97,7 +97,7 @@ function resetError() {
           {{ $t('msg.info.signUp.superAdmin') }}
         </h2>
 
-        <a-form ref="formValidator" :model="form" layout="vertical" no-style @finish="signUp">
+        <a-form v-if="appInfo.oidcAuthName == ''" ref="formValidator" :model="form" layout="vertical" no-style @finish="signUp">
```

Comment:
Relating to the comment above, if you use `null` as the default you can replace this with `v-if="!appInfo.oidcAuthName"`